example/layers.js

```diff
@@ -229,5 +229,39 @@
     var leafletControl = new L.control.layers(baseMaps,overlayMaps);
     leafletControl.addTo(map);
     
+    // Display number of active overlay layers
+    var touched = false;
+    var activeOverlays = leafletControl.getActiveOverlayNames();
+    var badge = L.control.badge(activeOverlays.length, { position: 'topright' }).addTo(map);
 
-    
+    map.on('layeradd', function(ev) {
```

Comment:
Lets also add all this code in the /src folder . We want this feature to work when user uses our allLayers feature which  automatically adds the menu bar. Makes sense? 
This file : https://github.com/publiclab/leaflet-environmental-layers/blob/master/src/AllLayers.js gives the user the ability to add multiple layers. Lets add here also. Thanks!

Reply:
Let's create a function of this and call here in example/layers.js and here https://github.com/publiclab/leaflet-environmental-layers/blob/master/src/AllLayers.js .